docs/modules/servers/pages/local/conf/index.adoc

```diff
@@ -0,0 +1,23 @@
+= James Local Mail Server Configuration
+:navtitle: Configuration
+
+ * Domain names (domainlist.xml)
+ * Database (james-database.properties)
+ * LMTP (lmtpserver.xml)
+ * POP3 (pop3server.xml)
+ * DNS (dnsservice.xml)
+ * Loggin (logback.xml)
+ * ??? (jmx.properties)
+ * ??? (recipientrewritetable.xml)
+ * ??? (extensions.properties)
+ * JWT (jwt_publickey)
+ * Mailets (mailetcontainer.xml)
+ * SMTP (smtpserver.xml)
+ * Health checks (healthcheck.properties)
+ * Mail storage (mailrepositorystore.xml)
+ * User storage (usersrepository.xml)
+ * IMAP (imapserver.xml)
+ * ??? (listeners.xml)
```

Comment:
Mailbox listeners allows to react upon users mailbox events like "message added", "message deleted"...

Reply:
Thanks!

Reply:
Sorry, are these the same as SMTP Hooks that are [described on the old website](https://james.apache.org/server/dev-provided-smtp-hooks.html)?

Where can I find examples of these?

Reply:
No it is different.

SMTP hooks are to add functionalities to the SMTP protocols.

Here MailboxListener allow to react upon any mailbox changes whatever the protocol.

See http://james.apache.org/server/config-listeners.html

Reply:
Ah, hehe. Ok, makes sense. I read the page at the url you pasted.

Are there any examples anywhere?

Reply:
If you look at example of users in the code, the quota-search feature is a good example : https://github.com/apache/james-project/blob/master/mailbox/plugin/quota-search-elasticsearch/src/main/java/org/apache/james/quota/search/elasticsearch/events/ElasticSearchQuotaMailboxListener.java

It allows to index quota usage per user into ElasticSearch

Reply:
Added above as a note